docs/UserGuide.md

```diff
@@ -93,69 +97,126 @@ Example:
 
 Deletes the specified patient from MediBook.
 
-Format: `delete <index>`
+####Format: 
 
-Examples:
+`delete INDEX`
+
+####Example:
 * `delete 1` deletes the patient with index `1` in the displayed list
 
 ### 3.6 Finding a patient by IC: `find`
 
 Finds patient records by multiple fields and multiple keywords.
 
+####Format: 
 
-Format: `find [i/IC] [n/NAME] [d/DATE_OF_BIRTH] [p/PHONE_NUMBER] [e/EMAIL] [a/ADDRESS] [h/HEIGHT] [w/WEIGHT] [b/BLOOD_TYPE]`
+`find [i/IC] [n/NAME] [d/DATE_OF_BIRTH] [p/PHONE_NUMBER] [e/EMAIL] [a/ADDRESS] [h/HEIGHT] [w/WEIGHT] [b/BLOOD_TYPE]`
 
 * At least one field must be specified for the command to be valid
 * The search is case-insensitive. e.g `S9123456A` will match `s9123456a`
 * The search will match substrings e.g `S912345` will match `S9123456A`
 * Search will return all records that matches any of the keywords for each field e.g `n/Alice Bob` will return the records of all patients with name containing `Alice` or `Bob`
 
-Examples: 
+####Examples: 
 * `find i/S9123456A` returns the patient record with IC `S9123456A`
 * `find n/Billy Alice a/Clementi i/S99` returns the patient records with Name containing `Billy` or `Alice`, Address containing `Clementi` and IC containing `S99`
 
 ### 3.7 Accessing a patient's profile : `access`
 
 Accesses a specified patient's profile in MediBook.
 
-Format: `access <index>`
+####Format: 
+
+`access INDEX`
 
 * Use the `list` command to return to the main list page
 * Each patient's profile consists of 3 sections: Personal Details, Medical Details and Medical Notes.
     * Personal Details section (top left): Shows the personal details of the patient, including their Name, IC, Date of Birth, and Phone Number, as well as their Email, Address, Height, Weight, BMI and Blood Type if available.
     * Medical Details section (bottom left): Shows medical details as tags separated into 3 categories (to be implemented), namely: Allergies, (ongoing) Treatments and (preexisting) Conditions.
     * Medical Notes section (right): A scrollable panel showing the list of medical notes recorded for the specified patient.
 
-### 3.8 Adding a medical note: `note`
+### 3.8 Adding a medical note: `addnote`
 
 Adds a medical note to a patient.
 
-As a doctor, you can add a medical note to a patient when viewing his/her profile. MediBook will automatically add the 
-to the patient displayed on the screen.
+As a doctor, you can add a medical note to your patient when viewing his/her profile to keep track of consultations
+with your patients. MediBook will automatically add the medical note to the patient displayed on your screen.
```

Comment:
Should this be a user story in the UG? May not be relevant to some readers of the UG. Is there a better way to phrase this/refer to doctors themselves? i.e. If you are a doctor...

Reply:
Ok I have updated this.